sdk/go/common/workspace/creds.go

```diff
@@ -133,13 +133,12 @@ func getCredsFilePath() (string, error) {
 // GetCurrentCloudURL returns the URL of the cloud we are currently connected to. This may be empty if we
 // have not logged in.
 func GetCurrentCloudURL() (string, error) {
-	var url string
-
 	// Allow PULUMI_BACKEND_URL to override the current cloud URL selection
 	if backend := os.Getenv(PulumiBackendURLEnvVar); backend != "" {
-		url = backend
+		return backend, nil
```

Comment:
This was introduced in https://github.com/pulumi/pulumi/pull/5789 however, this change was needed to use a local filesystem backend for the kubernetes operator.

Reply:
I thought we had tests covering this. If we did, was there a case missing?  If we didn't, worth adding them as part of this PR?

Reply:
> I thought we had tests covering this. If we did, was there a case missing? If we didn't, worth adding them as part of this PR?

I didn't see coverage for this particular code path. Will add now.

Reply:
Added test case.